### ISSUE TITLE
Can we clarify things around shortcuts and simple representations ?

### ISSUE THREAD

**Issue by pierrechoffe** [DOREMUS-ANR/doremus-ontology#22]
@rtroncy you recently said to the nice folk at Erlangen that "we love shortcuts". Can we clarify when shortcuts are useful and how to model and implement them ?

A simple example is u24_has_award. To state that a CD received a Diapason d'Or, we model it through a long path with M34_Award_Assignment, [see schema](https://drive.google.com/file/d/0B4s5dBwAmhJsZVpSbVdIUGdoc0U/view?usp=sharing).

But you can see in this schema that we also created u24_has_award : 
F24_Publication_Expression _u24_has_award_ M36_Award "diapason d'or"

The "long path" gives us more info, e.g. the author of the award ("Diapason"), the year, place.

The shortcut just gives us a handy way to answer the simple question "what awards did this CD receive" and also "which CDs received a Diapason". If we want to add a year in our request, we need the long path, so u24 is not a real shortcut, it's a _simpler representation_, hence the broad title of this issue.

(Yet, I use the word "shortcut" because on a similar case, _R40_has_representative_expression_,  the FRBRoo documentation speaks about shortcut for F1 - R50i - F42 - R41 - F2)

So : 
- what is the right definition of a shortcut ?
- how do we implement it ?
- is it ok to create simple representations that kind of double the long path ?

**Comment by pierrechoffe** [comment]
@rtroncy here is an example of a *simpler representation* together with the long path.

[This schema](https://drive.google.com/file/d/0B4s5dBwAmhJsSTlaOVNPbzhiWDA/view?usp=sharing) shows the Derivation Type Assignment we decided together on page 1.
On page 2, I added a "simpler" property : Uxx_is_derivation_type_of.
We may not want to ask for "all the possible derivation types of a Work", since it can only have one per institution. OK, institutions may have different points of view, and this query may highlight this, but this is a very limited option. Therefore, the property is oriented the other way around : "what works are (transcription, reduction, etc.)".

So my question is two-fold :
- is this property useful ?
- is its implementation correct ? (it is not implemented as a real shortcut but as a simpler property - simpler in the sense that we lose information : author, date, place)

**Comment by pierrechoffe** [comment]
@rtroncy I come back to you regarding "shortcuts". Since we may have a number of shortcuts it would be nice to have a clear view on this topic. For me there are 2 ways we could consider a shortcut, either as a chain of properties or as a distinct, independent property.

Therefore I have 2 distinct questions : 

* Is it ok to have both types of shortcuts? does it make sense? is it useful? Or should we do with one type only, i.e. chained properties ?
* how do we implement chained properties? how do we create them in Protégé?

Thx

**Comment by rtroncy** [comment]
As I briefly replied to you in Montpellier, this is not for now and you should not be too much concerned about it. If you need a short direct property, just create it. We will always be able to strengthen the definition of a property by adding equivalence axioms composed of property chains. We should aim to release **asap** the version 1 of the DOREMUS ontology and such axioms can be part of future releases. I have assigned myself this action and I will keep it open, working on it after the release of the first version of the ontology

**Comment by rtroncy** [comment]
Not really: we need an inventory of the shortcuts we have defined and implemented (i.e. using a single property instead of a longer path) so that we implement the corresponding property chain axioms. Perhaps something to discuss in Montpellier? I'm adding it to the agenda.

**Comment by pasqLisena** [comment]
> we need an inventory of the shortcuts we have defined and implemented

For sure, none of them are implemented. I am not sure if we defined any actually.

**Comment by pasqLisena** [comment]
> ACTION: Inventorier tous ces cas (à la fois par le groupe modélisation via le tableau de mapping et côté marc2rdf)
from https://docs.google.com/document/d/1cekU4np8aRWSWSKVBPQSftJOTNSPqoj8LchnDyGJXQo/edit#heading=h.d9mw6z690xu
